sdk/src/test/java/com/google/cloud/dataflow/sdk/io/FileBasedSinkTest.java

```diff
@@ -248,20 +248,20 @@ private void testRemoveTemporaryFiles(int numFiles, String baseTemporaryFilename
     PipelineOptions options = PipelineOptionsFactory.create();
     SimpleSink.SimpleWriteOperation writeOp = buildWriteOperation(baseTemporaryFilename);
 
-    List<File> temporaryFiles = new ArrayList<>();
+    List<String> temporaryFiles = new ArrayList<>();
     List<File> outputFiles = new ArrayList<>();
     for (int i = 0; i < numFiles; i++) {
```

Comment:
Please add a unit test that specifically passes in temporary files that would have never been matched.